assets/sass/custom.scss

```diff
@@ -1,23 +1,40 @@
+// Had an error not loading variables 
+@import "../../node_modules/bulma/sass/utilities/_all";
```

Comment:
This is already imported and removing it does not cause any error for me.

Reply:
I had an error for `@imports` with variables from Bulma.

Reply:
Strange, can you repro it? I can't as it's already included in assets/sass/syle.scss which includes this file.